src/modules/postgres/tests/query.test.js

```diff
@@ -0,0 +1,49 @@
+import test from 'ava';
+import query from '../query';
+
+test('bounds query', t => {
+	t.plan(1);
```

Comment:
unnecessary to use `t.plan` here. use `t.plan` only when there are thinke callbacks and loops where you need to ensure that a certain number of assertions happen.